public/index.html

```diff
@@ -14,13 +14,24 @@
     <script>
       // Jaeger UI config data is embedded by the query-service. This is
       // later merged with defaults into the redux `state.config` via
-      // src/utils/config/get-config.js. The default provided by the query
-      // service should be an empty object or it can leave `DEFAULT_CONFIG`
+      // src/utils/config#getUiConfig. The default provided by the query
+      // service should be an empty object or it can leave `DEFAULT_UI_CONFIG`
       // unchanged.
       function getJaegerUiConfig() {
-        const DEFAULT_CONFIG = null;
-        const JAEGER_CONFIG = DEFAULT_CONFIG;
-        return JAEGER_CONFIG;
+        const DEFAULT_UI_CONFIG = null;
+        const JAEGER_UI_CONFIG = DEFAULT_UI_CONFIG;
```

Comment:
why do we need DEFAULT_ constants? When I originally proposed them I thought they would have some concrete values, and it was easier to search&replace the string `JAEGER_UI_CONFIG = DEFAULT_UI_CONFIG` than to replace the JS expression on the right hand side. But these are always null anyway, we might as well have `JAEGER_UI_CONFIG = null`, it's just as easy to search&replace.

Reply:
We could do that.

I think tweaking the name a bit to imply it might be over-written has some value, though:

```javascript
// use `null` as a fallback value if the query-service does not embed a value
const __INJECTED_UI_CONFIG__ = null;
const JAEGER_UI_CONFIG = __INJECTED_UI_CONFIG__;
```

Reply:
+1 for `__INJECTED_UI_CONFIG__`. I don't think you need the second const, you can simply `return __INJECTED_UI_CONFIG__;`.

Reply:
Great. Done.